AGS_Adapter/AGSAdapter.cs

```diff
@@ -37,32 +39,65 @@ public partial class AGSAdapter : BHoMAdapter
         /**** Constructors                              ****/
         /***************************************************/
 
-        [Description("Adapter for AGS.")]
+        [Description("Adapter for AGS files.")]
         [Output("The created AGS adapter.")]
-        public AGSAdapter()
+        public AGSAdapter(string filePath, AGSConfig agsConfig = null, bool active = false)
```

Comment:
I would recommend using the `BH.oM.Adapter.FileSettings` object for consistency with other adapters (at some point, not right now).